docs/devguides/yandex_dialogs.rst

```diff
@@ -0,0 +1,20 @@
+Yandex.Dialogs integration
+==========================
+
+
+Different parts of DeepPavlov library could be launched as an endpoint for Yandex.Dialog platform.
```

Comment:
I think we should also include a link on [yandex documentation on that](https://tech.yandex.ru/dialogs/alice/doc/about-docpage/)

Reply:
You can commit anything you want - I am ok with that. I'll not be able to make any changes before tomorrow afternoon.